SparkPost/2web2ui
# AC-1715 Replace occurrences of api.segment.io and cdn.segment.com with Sparkpost proxy

### What Changed

- Changed the single instance of `cdn.segment.com` with our proxy domain `cdn.analytics.sparkpost.com`
  - When the CDN is loaded on the page, it will make network requests to `api.analytics.sparkpost.com` instead of `api.segment.io`
- Added both of the proxy URLs to our `blockHosts` list for cypress tests (with a comment for clarity of what it is)

### How To Test

1. Load any page in the app
2. Open network request tab
3. Search for "segment", ensure there are no calls to either cdn.segment.com or api.segment.io
4. Search for "analytics.sparkpost.com", ensure there is at least a call to `cdn.analytics.sparkpost.com` and `api.analytics.sparkpost.com`.

1. Turn on adblocker (ublock, for example)
2. Repeat all of the previous steps and ensure none of those requests are being blocked by default.


### Findings
- We were unsure how Segment was changing their backend to accommodate our use of `api.analytics.sparkpost.com` since we do not manually use their original `api.segment.io` domain in our code.
  - They used a script to update the CDN call _for each source_ that replaced it with our proxy domain.
